src/scheduling/appointments/view/ViewAppointment.tsx

```diff
@@ -14,7 +14,7 @@ import useAddBreadcrumbs from '../../../breadcrumbs/useAddBreadcrumbs'
 
 const ViewAppointment = () => {
   const { t } = useTranslation()
-  useTitle(t('scheduling.appointments.viewAppointment'))
+  useTitle(t('Appointments'))
```

Comment:
the string that is passed to the translation function needs to be a key defined in the translation files.

I would expect that this PR would not change this line of code, but would rather add a key in the translation files that allow for scheduling.appointments.viewAppointment to map to the string View Appointment.